bufferserver/src/main/java/com/datatorrent/bufferserver/internal/DataList.java

```diff
@@ -845,6 +854,30 @@ protected void release(boolean wait)
       }
     }
 
+    private boolean canRelease(boolean writer)
```

Comment:
What do you think about renaming release() to checkRelease() instead? If not, I can change canRelease() it to canEvict()?